CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
```

Comment:
This is missing a license header (this is the reason why the CI fails).

Reply:
Done.

Reply:
Did you really write this in 2021 and waited till now to submit the PR? 🙂